api/services/sns.js

```diff
@@ -0,0 +1,26 @@
+const AWS = require('aws-sdk');
+const Promise = require('bluebird');
+const sns = new AWS.SNS({
+  accessKeyId: sails.config.globals.accessKeyId,
+  secretAccessKey: sails.config.globals.secretAccessKey,
+  region: sails.config.globals.serviceRegion,
+});
+Promise.promisifyAll(sns);
```

Comment:
Doesn't look like you need to promisify the `sns` instance since you're not using the async version of the `publish` method.